src/transformers/pipelines.py

```diff
@@ -1284,6 +1284,9 @@ def __call__(self, *args, **kwargs):
             min_null_score = 1000000  # large and positive
             answers = []
             for (feature, start_, end_) in zip(features, start, end):
+                # Ensure padding cannot be attended
```

Comment:
Maybe improve comment to => "cannot be attended" makes me think of preparing inputs to a model to not be attended by the self-attention layers, but `start` and `end` are the outputs of the model.

Do we even need the lines `start_, end_ = start_ * feature.attention_mask, end_ * feature.attention_mask`? 
Can't we just set all index whose `attention_mask` is 0 to -10000 later?

Reply:
Otherwise maybe just change comment to `Zero out start and end logits at masked positions`